src/Trips.js

```diff
@@ -1,14 +1,15 @@
 class Trip {
-  constructor() {
-    this.id = id;
-    this.userID = userID;
-    this.destinationID = destinationID;
-    this.travelers = travelers;
-    this.date = date;
-    this.duration = duration;
-    this.status = status;
-    this.suggestedActivities = suggestedActivities;
+  constructor(data) {
+    this.data = data;
```

Comment:
i think it's fine here to pass in a single parameter (tripData would be more declarative), but i think you should still have the individual properties separated out, like "this.id = tripData.id, this.userID = tripData.userID". That way, you don't have to use "trip.data.id" or "trip.data.duration" every time you want to access a property.

Reply:
Thanks. I think I avoid this because I was reprimanded in my Mod 1 assessment. I think it makes the most sense in this case.